### PR TITLE
Add dynamic container and skeleton loader

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue [hunk 1]
```diff
@@ -19,14 +19,24 @@
               class="white">
               <v-flex d-flex xs12>
                 <v-card flat>
-                  <v-card-text class="subtitle-2 grey--text pa-2">{{ this.$t('exoplatform.perkstore.title.myOrders') }}</v-card-text>
+                  <v-card-text class="subtitle-2 grey--text pa-2">
+                    <span :class="firstLoadingPerkStoreOrder && 'skeleton-text skeleton-background skeleton-header skeleton-border-radius'">
+                      {{ this.$t('exoplatform.perkstore.title.myOrders') }}
```
Comment (boubaker) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue]: no usage of `this.` inside Vue template, you can use `$t('exoplatform.perkstore.title.myOrders')`

File: perk-store-webapps/src/main/webapp/perkstoreOrder.html [hunk 2]
```diff
@@ -5,5 +5,30 @@
...
+            <div class="v-application--wrap">
+                <main>
+                    <div class="container pa-0">
+                        <div class="layout row wrap mx-0" style="cursor: pointer;">
+                            <div class="flex d-flex sx12">
+                                <div class="layout white row ma-0">
+                                    <div class="flex d-flex xs12">
+                                        <div class="v-card v-card--flat v-sheet theme--light">
+                                            <div class="v-card__text subtitle-2 grey--text pa-2"><span class="skeleton-text skeleton-background skeleton-header skeleton-border-radius">
+                              Mes commandes
```
Comment (boubaker) [inline perk-store-webapps/src/main/webapp/perkstoreOrder.html]: Replace this litteral translated string by `&nbsp;`
(same for other strings)

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue (hunk 1 above)
Comment (SaraBoutej) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue]: Done!

File: perk-store-webapps/src/main/webapp/perkstoreOrder.html (hunk 2 above)
Comment (SaraBoutej) [reply perk-store-webapps/src/main/webapp/perkstoreOrder.html]: Done!

**Review by boubaker — DISMISSED** [review]

**Review by boubaker — APPROVED** [review]

**Pull request merged**